src/app/routes/article/getInitialData/index.ts

```diff
@@ -12,9 +13,11 @@ import { Services, Variants } from '../../../models/types/global';
 const logger = nodeLogger(__filename);
 
 const removeAmp = (path: string) => path.split('.')[0];
-const popId = (path: string) => path.match(/(c[a-zA-Z0-9]{10}o)/)?.[1];
+const getOptimoId = (path: string) => path.match(/(c[a-zA-Z0-9]{10}o)/)?.[1];
```

Comment:
reminds me - at some point in 2023 we should correct this regex to allow for more than 10 chars + c & o...

Reply:
not important now, though

Reply:
Yea definitely worth changing this soonish. I'll draft up a ticket on this as it needs updated in a few places.